Makefile

```diff
@@ -80,6 +82,13 @@ $(SVC_APPLIED): apply-service-%: ci/ups/$(CLD_HOST)/%.json
 $(SVC_CREATED): create-service-%:
 	$(MAKE) apply-service-$* APPLY_ACTION=create
 
+$(SVC_DIFFED): SHELL = /bin/bash
+$(SVC_DIFFED): diff-service-%:
+	@diff \
```

Comment:
@dagoof could we use `sdiff` or `diff -s`? More useful output IMO.

Reply:
tried sdiff, why is it more useful here? instead of listing the changes mine is now echoing out both identical files